entry_tasks/86_create_phone_book/86_create_phone_book.py

```diff
@@ -0,0 +1,12 @@
+if __name__ == '__main__':
+    friends_total = int(input('How many friends do you have?: '))
+    phone_book = dict()
+    for n in range(friends_total):
+        name = input("Please enter name of friend ({}): ".format(n + 1))
+        phone = input("Please enter {}'s phone: ".format(name))
+        print('Contact "{}" with phone "{}" created.'.format(name, phone))
+        phone_book_input = {name: phone}
```

Comment:
why we can't just set new key in phone_book directly and create extra dict instead? Our memory wont like it!